### ISSUE TITLE
Set importer metadata source as a item/album field 

### ISSUE THREAD

**Issue by kooimens** [sampsyo/beets#1311]
I'm using three types of sources in my library: MB, Discogs or none (imported asis). The ability to list/modify etc. albums or items that came from a particular source would be useful.

**Comment by sampsyo** [comment]
Seems like a good idea! I could have sworn we'd talked about this before, but I can't find a previous issue either.

**Comment by kooimens** [comment]
I was thinking the same, but didn't find anything similar. For now, I think I found a workaround. Figured out that the mb_albumartistid is filled with a number (e.g. 255410) if the release is from discogs. If it's from MB it looks a bit like `4de9f2e9-164b-4c0a-9573...`. In conclusion this means that the mb_albumartistid is at least 15 characters long if the release is from MB and probably below that value if it's from discogs. For items that are tagged asis the length will be zero (not always the case if you are having a messy library). By using the inline plugin I set the `source` tag like this:
`source: 1 if len(mb_albumartistid) < 15 and len(mb_albumartistid) > 0 else 2 if len(mb_albumartistid) == 0 else 0`
Not perfect, but it works. Maybe others can use this as well.
